### PR TITLE
Update runtime spec to rc5

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by mrunalp** [comment]
@crosbymichael PTAL

**Comment by crosbymichael** [comment]
@justincormack can you double check the cap changes

**Comment by crosbymichael** [comment]
LGTM

**Comment by crosbymichael** [comment]
Do we need to remove the `capabilities_ambient.go` file and build tag now what we have support for it built into the spec?

File: libcontainer/configs/config.go [hunk 1]
```diff
@@ -114,7 +114,7 @@ type Config struct {
 
 	// Capabilities specify the capabilities to keep when executing the process inside the container
 	// All capbilities not specified will be dropped from the processes capability mask
```
Comment (justincormack) [inline libcontainer/configs/config.go]: may as well fix the typo above too...

**Comment by justincormack** [comment]
Yes, the build tag needs to go.

**Comment by mrunalp** [comment]
Pushed a commit to remove the ambient build tag.

File: libcontainer/configs/config.go (hunk 1 above)
Comment (mrunalp) [inline libcontainer/configs/config.go]: Fixed

**Comment by crosbymichael** [comment]
LGTM

File: vendor.conf [hunk 2]
```diff
@@ -6,7 +6,7 @@ github.com/docker/go-units 9b001659dd36225e356b4467c465d732e745f53d
 github.com/godbus/dbus c7fdd8b5cd55e87b4e1f4e372cdb1db61dd6c66f
 github.com/golang/protobuf/proto f7137ae6b19afbfd61a94b746fda3b3fe0491874
 github.com/mrunalp/fileutils ed869b029674c0e9ce4c0dfa781405c2d9946d08
-github.com/opencontainers/runtime-spec/specs-go 794ca7ac88234607f9d2c76da8a6e9bbbade8cb9
+github.com/opencontainers/runtime-spec/specs-go 035da1dca3dfbb00d752eb58b0b158d6129f3776
```
Comment (cyphar) [inline vendor.conf]: I'd prefer if we used tags here...

**Comment by cyphar** [comment]
LGTM, aside from the nit. Feel free to merge.

**Comment by dqminh** [comment]
LGTM

**Pull request merged**